Client/src/Components/Sidebar/index.jsx

```diff
@@ -173,32 +173,39 @@ function Sidebar() {
       }}
     >
       <Stack pt={theme.spacing(6)} pb={theme.spacing(12)} pl={theme.spacing(8)}>
-        <Stack direction="row" alignItems="center" gap={theme.spacing(4)}>
-          <Stack
-            justifyContent="center"
-            alignItems="center"
-            minWidth={theme.spacing(16)}
-            minHeight={theme.spacing(16)}
-            pl="1px"
-            fontSize={18}
-            color="white"
-            sx={{
-              position: "relative",
-              backgroundColor: theme.palette.primary.main,
-              borderRadius: theme.shape.borderRadius,
-              userSelect: "none",
-            }}
-          >
-            BU
+        <a 
+          href="https://github.com/bluewave-labs/bluewave-uptime" 
+          target="_blank" 
+          rel="noopener noreferrer" 
+          style={{ textDecoration: "none", color: "inherit" }}
+        >
```

Comment:
This works, but we can simplify and dispense with the anchor tag all together.

All we have to do is set an `onClick` listener ont he stack:

```
<Stack direction="row" alignItems="center" gap={theme.spacing(4)} onClick={() => {// your action here}}>
```

and then you can call

```
window.open("http://example.com", "_blank", "noreferrer");
```

in the `onClick` listener

That will allow us to keep the DOM as simple as possible which is always a pluss :laughing: